byron/ledger/formal-spec/update-mechanism.tex

```diff
@@ -130,6 +132,7 @@ \subsection{Update proposals}
       \fun{upSwVer} & \UProp \to \SWVer & \text{software-version update proposal}\\
       \fun{upSig} & \UProp \to \Sig & \text{update proposal signature}\\
       \fun{upSigdata} & \UProp \to \UPropSD & \text{update proposal signed data}\\
+      \fun{upSTags} & \powerset{\STag} & \text{update proposal system tags}
```

Comment:
Might be nice to have some more definition of what this is in prose.

Reply:
Good point. Done.